components/post_view/channel_intro_message/__snapshots__/channel_intro_message.test.tsx.snap

```diff
@@ -40,54 +55,31 @@ exports[`components/post_view/ChannelIntroMessages test DEFAULT Channel should m
     }
     teamId="team-id"
   >
-    <Connect(injectIntl(ToggleModalButton))
-      ariaLabel="Set a Header"
-      className="intro-links color--link channelIntroButton"
-      dialogProps={
-        Object {
-          "channel": Object {
-            "create_at": 1508265709607,
-            "creator_id": "creator_id",
-            "delete_at": 0,
-            "display_name": "test channel",
-            "header": "test",
-            "id": "channel_id",
-            "last_post_at": 1508265709635,
-            "name": "town-square",
-            "purpose": "test",
-            "team_id": "team-id",
-            "type": "O",
-            "update_at": 1508265709607,
-          },
-        }
-      }
-      dialogType={
+    <Memo(SetHeaderButton)
```

Comment:
`SetHeaderButton` should always replace the `ToggleModalButton` in these tests because no tests exist that include an archived channel. 

Need to add at least one test that includes and archived channel